libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -85,6 +86,25 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // @Values: -1:Disabled,0:Bus0:1:Bus1
     // @User: Advanced
     AP_GROUPINFO("EXT_BUS", 7, AP_Baro, _ext_bus, -1),
+
+    // @Param: SPEC_GRAV
+    // @DisplayName: Specific Gravity (For water depth measurement)
+    // @Description: This sets the specific gravity of the fluid when flying an underwater ROV. Set to 1.0 for freshwater or 1.024 for saltwater
```

Comment:
I was actually saying the contrary - remove it from the description. Values are valuable to be shown in GCSs, like in a combobox for example.